### ISSUE TITLE
0.6.9 fails to load sensor

### ISSUE THREAD

**Issue by sebr** [twrecked/hass-aarlo#157]
```
2020-01-24 09:42:45 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 284, in async_update_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 328, in _async_write_ha_state
    attr.update(self.device_state_attributes or {})
  File "/config/custom_components/aarlo/sensor.py", line 139, in device_state_attributes
    attrs['camera_name'] = self._device.name
AttributeError: 'PyArlo' object has no attribute 'name'
```

**Comment by twrecked** [comment]
Sorry about that. Should be fixed in 0.6.9.1.

**Comment by sebr** [comment]
Thanks, works now!
